### DISCUSSION TITLE
addComponents - storage  behaviour

### DISCUSSION THREAD

**Discussion by TarikNasraoui** [GrapesJS/grapesjs#4300] [Q&A]
Hi,
When I add a Components with editor.addComponents() and storage is enabled then the component is added at each reload
here is an example: 

editor.addComponents(`<div>
  <span `title="foo">Hello` world!!!</span></div>`);

storageManager: {
      id: "gjs-",
      type: "local",
      autosave: true,
      storeComponents: true,
      storeStyles: true,
      storeHtml: true,
      storeCss: true,
    },
<img width="786" alt="Capture d’écran 2022-04-29 à 15 03 18" src="https://user-images.githubusercontent.com/30218125/165949872-5cbf471a-2bbc-4bd3-8fa6-661404a92ba0.png">

**Comment by artf** [comment]
Well yeah, if you keep adding content on each reload, the storage will keep saving it...
